voice_assistant_pkg/README.md

```diff
@@ -0,0 +1,173 @@
+# 1. How I created the packages **"servo_interfaces"** and **"voice_assistant_pkg"**
+
+The instructions in this section are only for bookkeeping how I created the packages. Follow section [2](#2-how-to-build-and-run-the-packages) to build and run packages.
+## 1.1. Create **"servo_interfaces"** package to host RunServo.srv
```

Comment:
This is really confusing, why are the servo command for hte snack dispenser and the voice control of the robot in the same package, in a mixed readme?!

Reply:
This is because I naively dumped everything in one package so that we can discuss the best way to organize the code in this PR :D